reconcile/cna/client.py

```diff
@@ -23,15 +70,44 @@ def list_assets(self) -> list[dict[str, Any]]:
         cnas = self._ocm_client.get(api_path="/api/cna-management/v1/cnas")
         return cnas.get("items", [])
 
+    def fetch_bindings_for_asset(self, asset: Asset) -> list[dict[str, str]]:
```

Comment:
Generally speaking, what's the failure mode for this client? Do we raise an exception for 5xx? What's the retry strategy? Maybe overkill to document it for each method, but if the behavior is consistent then maybe added it to the `CNAClient` docstring?

If this is ultimately all driven by `OCMBaseClient`, then maybe that can be stated as well?

Reply:
i added a comment to the class doc string - [443690b](https://github.com/app-sre/qontract-reconcile/pull/3056/commits/443690b2af95f5b2b9ee7a07e6c68596aa7cb62d)